.github/workflows/macos.yml

```diff
@@ -49,7 +48,12 @@ jobs:
           path: ${{ env.logdir_path }}/*
           retention-days: 3
           overwrite: true
-        if: always()
+        if: contains(env.logdir_path, 'kDrive-logdir')
```

Comment:
Would it be possible to handle the exceptional situation where the the log directory does not exist? 
An error or warning message could be issued. Would the build fail in this case?

Reply:
In case the logdir is not found, the Cleanup step is skipped and when looking at the run info there will be a specific `skipped` icon displayed